api/src/controllers/signin.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const router = express.Router();
+const Joi = require("joi");
+const jwt = require("jsonwebtoken");
+const { getToken } = require("../passport");
+const config = require("../config");
+const { serializeYoung, serializeReferent } = require("../utils/serializer");
+const { ERRORS } = require("../utils");
+
+const Young = require("../models/young");
+const Referent = require("../models/referent");
+const { capture } = require("../sentry");
+
+router.get("/signin_token", async (req, res) => {
```

Comment:
Voir commentaire précédent

```suggestion
router.get("/token", async (req, res) => {
```